README.md

```diff
@@ -42,7 +47,111 @@ Specification](https://adobe-type-tools.github.io/font-tech-notes/pdfs/5176.CFF.
+- [libint10h_fonts](https://github.com/cellularmitosis/libint10h_fonts) - The int10h.org bitmap font collection as a library of C structs.
+- [ttf2mesh](https://github.com/fetisov/ttf2mesh) - Standalone library for TrueType font tessellation. Allows to load ttf-file and convert its glyphs to 2D or 3D mesh objects without rasterization.
+- [otf2ttf](https://codeberg.org/zzkt/otf2ttf) - convert an OTF font to TTF.
+- [psftools](https://codeberg.org/gnarz/psftools) - a simple textfile based psf font editor suite.
+- [font-config-info](https://codeberg.org/derat/font-config-info) - Print Linux font configuration.
+
+## C++
+- [font to svg](https://github.com/donbright/font_to_svg) - Render characters from font files into an SVG path.
+- [LCDF Typetools](https://github.com/kohler/lcdf-typetools) - Utilities for manipulating OpenType, PostScript Type 1, and Multiple Master fonts.
+- [HarfBuzz](https://github.com/googlei18n/harfbuzz) - HarfBuzz OpenType text shaping engine.
+- [FontView](https://github.com/googlei18n/fontview) - Demo app that displays fonts with a free/libre/open-source text rendering stack: FreeType, HarfBuzz and Raqm.
+- [Text rendering tests](https://github.com/unicode-org/text-rendering-tests) - Test suite for text rendering.
+- [FTGLES](https://github.com/cdave1/ftgles) - TrueType font rendering library for OpenGL ES on iOS devices (iPad and iPhone).
+- [TTFPatch](https://github.com/rmuch/ttfpatch) - Mirror of TTFPATCH by Wolfram Esser, modified to support fonts conforming to newer OTF specifications, built for modern versions of Windows.
+- [stb-truetype-opengl-examples](https://github.com/0xc0dec/demos) - Examples of TrueType font rendering in C++11 using stb_truetype library and OpenGL 3+.
+- [WOFF2](https://github.com/google/woff2)
+- [vectorfontstorm](https://github.com/VoxelStorm-Ltd/vectorfontstorm) - C++ vector font rendering library for OpenGL 3D text rendering, by VoxelStorm Resources
+- [Slug library](https://sluglibrary.com/) - Dynamic GPU Font Rendering and Advanced Text Layout
+- [trex](https://github.com/KyrietS/trex) - Font rendering, atlas generation and text shaping library written in C++
+- [ascii-art](https://github.com/codewithnick/ascii-art) - A C++ library to make everyday alphabets look much better on the terminal , this project uses OOPS concepts to make unique fonts and display letters on command line.
+- [DigiFont](https://github.com/cbm80amiga/DigiFont) - Library for nice, scalable digit rendering using any Arduino GFX libraries.
+- [artery-font-format](https://github.com/Chlumsky/artery-font-format) - Artery Atlas Font format library.
+- [bmfont](https://sourceforge.net/projects/bmfont) - Bitmap font generator.
+- [psfmaker](https://codeberg.org/CodePoet2005/psfmaker) - Tool for making linux console fonts. Output psf files and c header files.
+- [bdfreader](https://codeberg.org/heu/bdf-reader) - BDF font format reader.
```

Comment:
I think that "bdfreader" should be moved to the Rust category

Reply:
Yes bdf-reader should be moved to the rust category